unraid_mcp/tools/keys.py

```
@@ -0,0 +1,149 @@
+
+        try:
+            logger.info(f"Executing unraid_keys action={action}")
+
+            if action == "list":
+                data = await make_graphql_request(QUERIES["list"])
+                keys = data.get("apiKeys", [])
+                return {"keys": list(keys) if isinstance(keys, list) else []}
+
+            if action == "get":
+                if not key_id:
+                    raise ToolError("key_id is required for 'get' action")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)